### PR TITLE
View components & environment flow while viewing details

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
## Fixes 
Fixes https://issues.redhat.com/browse/HAC-2151

## Description
Updates the application details view to use the environment cards as tabs to show the application environment details view below on the same page.

Updates PF versions to pick fix for topology view sizing

## Type of change

- [x] Feature

## Screen shots / Gifs for design review 
image

image

image

image

image

/cc @itsptk @serenamarie125 @christianvogt @karthikjeeyar

### CODE REVIEW

File: src/components/Environment/ApplicationEnvironmentCards.tsx [hunk 1]
```diff
@@ -74,58 +89,43 @@ const ApplicationEnvironmentCard: React.FC<ApplicationEnvironmentCardProps> = ({
...
+              {isPositionedEnvironment(env, environments.slice(0, index)) ? (
+                <div className="application-environment-cards__env-arrow">
+                  <ArrowRightIcon />
+                </div>
+              ) : null}
+              <ApplicationEnvironmentCard
+                environment={env}
+                isExpanded={cardsExpanded}
+                isSelected={env.metadata.name === selectedEnvironment}
+                onSelect={() => onSelect && onSelect(env.metadata.name)}
```
Comment (karthikjeeyar) [inline src/components/Environment/ApplicationEnvironmentCards.tsx:119]: we can use same displayName in the heading below like we have in card title.

image


```suggestion
                onSelect={() => onSelect && onSelect(env.spec.displayName)}
```

**Comment by karthikjeeyar** [comment]
When there is getting started card on the top of the page, then the graph area is too small to work with (height 150px). 
 
image

File: src/components/Environment/ApplicationEnvironmentCards.tsx (hunk 1 above)
Comment (rohitkrai03) [reply src/components/Environment/ApplicationEnvironmentCards.tsx:119]: Agreed, I think it would be better to use display name here as well.

**Review by rohitkrai03 — APPROVED** [review]
/approve

**Comment by rohitkrai03** [comment]
@jeff-phillips-18 Can you update the PR with the small change that @karthikjeeyar suggested? We can merge the PR after that.

**Comment by jeff-phillips-18** [comment]
@rohitkrai03 Updated and squashed. Thanks!

**Comment by jeff-phillips-18** [comment]
@karthikjeeyar I've reverted your requested change and updated only where we display the title to use the `spec.displayName`. Using the displayName for determining selection could be problematic since it is not guaranteed to be unique.

**Pull request merged**